backend/lib/edgehog/tenants/reconciler.ex

```diff
@@ -66,8 +66,8 @@ defmodule Edgehog.Tenants.Reconciler do
       schedule_reconciliation(@reconcile_interval)
     end
 
-    Tenants.list_tenants()
-    |> Tenants.preload_astarte_resources_for_tenant()
+    Tenant
+    |> Tenants.read(load: tenant_reconciliation_loads())
```

Comment:
Shoudn't it be `read!` to get list of tenants?
```suggestion
    |> Tenants.read!(load: tenant_reconciliation_loads())
```